native_client/README.md

```diff
@@ -67,15 +67,19 @@ ln -s ../DeepSpeech/native_client ./
 
 ## Building
 
-Before building the DeepSpeech client libraries, you will need to prepare your environment to configure and build TensorFlow. Follow the [instructions](https://www.tensorflow.org/install/install_sources) on the TensorFlow site for your platform, up to the end of 'Configure the installation'.
+Before building the DeepSpeech client libraries, you will need to prepare your environment to configure and build TensorFlow. 
+Preferably, checkout the version of tensorflow which is currently supported by DeepSpeech (see requirements.txt), and use bazel version 0.10.0. 
+Then, follow the [instructions](https://www.tensorflow.org/install/install_sources) on the TensorFlow site for your platform, up to the end of 'Configure the installation'.
 
-Then you can build the Tensorflow and DeepSpeech libraries. Please note that the flags for `libctc_decoder_with_kenlm.so` differs a little bit.
+After that, you can build the Tensorflow and DeepSpeech libraries using the following commands. Please note that the flags for `libctc_decoder_with_kenlm.so` differs a little bit.
 
 ```
-bazel build -c opt --copt=-O3 //native_client:libctc_decoder_with_kenlm.so
-bazel build --config=monolithic -c opt --copt=-O3 --copt=-fvisibility=hidden //native_client:libdeepspeech.so //native_client:deepspeech_utils //native_client:generate_trie
+bazel build --cxxopt="-D_GLIBCXX_USE_CXX11_ABI=0" -c opt --copt=-O3 //native_client:libctc_decoder_with_kenlm.so
```

Comment:
Sorry for missing this one, but `--copt=` should be better. And please make it the latest element :)

Reply:
Is it relevant to make it a C option, since it is about c++11 ?

Reply:
I prefer that we be safe and cover improper mixing of build flags. Bazel doc states that C-specific options should be set through `--conlyopt`, so we should be fine doing `--copt` :)